### PR TITLE
WIP: Versioned docs

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -679,7 +679,10 @@ docs:
 	@hash hugo > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
 		curl -sL https://github.com/gohugoio/hugo/releases/download/v0.74.3/hugo_0.74.3_Linux-64bit.tar.gz | tar zxf - -C /tmp && mv /tmp/hugo /usr/bin/hugo && chmod +x /usr/bin/hugo; \
 	fi
-	cd docs; make trans-copy clean build-offline;
+	cd docs; make trans-copy clean ;
+	#git co release/v1.13 && PUBLIC=public/v1.13 make build-offline ;
+	git co release/v1.14 && PUBLIC=public/v1.14 make build-offline ;
```
Comment (silverwind) [inline Makefile]: Can we avoid having to hardcode the release branch? Maybe some logic that builds all branches, back to a certain starting branch?

**Comment by wxiaoguang** [comment]
Some more users are misled by documents recently, they are asking about why they do not have the registry feature.

**Comment by silverwind** [comment]
I think the final solution for docs will be [docusaurus](https://docusaurus.io/), which has support for versioning.

**Comment by wxiaoguang** [comment]
Done is better than perfect, as long as it can continue to improve without difficulty.

If we can use a simple mechanism to split the documents for versions (months ago), then there will be no user asking about strange questions any more, and there is still a chance to improve the versioned document system.

Today there is one more user asking about why he doesn't package feature in discord.

**Comment by silverwind** [comment]
Yeah, I'm not opposed to having some intermediate solution now, just pointing out potential better solutions for the future :)

**Comment by wxiaoguang** [comment]
Today one more
* https://github.com/go-gitea/gitea/issues/19366

**Comment by wxiaoguang** [comment]
* https://github.com/go-gitea/gitea/issues/19549

**Pull request closed without merging**